storage/objects/main.go

```diff
@@ -424,6 +426,75 @@ func downloadUsingRequesterPays(client *storage.Client, object, bucketName, loca
 	return nil
 }
 
+func generateV4GetObjectSignedUrl(client *storage.Client, bucketName, objectName, serviceAccount string) (string, error) {
+	// [START storage_generate_signed_url_v4]
```

Comment:
Might be better to make new samples follow the new canonical sample format, putting these in their own file and having the region tag include imports. See #650.